pkg/testhelper/testhelper.go

```diff
@@ -106,6 +106,7 @@ const (
 	ReasonForCompliance             = "Reason For Compliance"
 	Category                        = "Category"
 	RoleBindingName                 = "Role Binding Name"
+	ClusterRoleBindingName          = "Cluster Role Reference Name"
```

Comment:
Should the name of the variable be changed as well?

`ClusterRoleBindingName` --> `ClusterRoleName`

Reply:
yah sure